docker/constraints.in

```diff
@@ -9,9 +9,9 @@ celery>=4,<5
 ciso8601
 click>=8.0
 cloudpickle>=0.4
-compliance-checker>=4.0.0
```

Comment:
FYI, compliance checker is only for checking NetCDF metadata compliance.

If we ever get to the point of making NetCDF support optional, it can be optional too.